test_unit.py

```diff
@@ -1241,46 +1241,44 @@ def test_periodic_payment(self):
 
-        golden_addr = ("WLTFMUFK2BRUOZTWKZIU5LUUR6FJZ" +
-                       "BFRJCDLMF35PXQEBDWSDT2CSGMK3M")
+        golden_addr = ("LXQWT2XLIVNFS54VTLR63UY5K6AMI" +
+                       "EWI7YTVE6LB4RWZDBZKH22ZO3S36I")
 
-        golden = ("ASAKAAEKAsDEBwSQTiSBAotqJgEgkq+RhOQTPAl/ZqvMk7ERGxKiAb2d" +
-                  "DMo+SkihzhPM9MUxFiISMRAjEhAxASQOEDIEIxJAAFUyBCUSMQghBA0Q" +
-                  "MQkyAxIQMwEQIQUSEDMBESEGEhAzARQoEhAzARMyAxIQMwESIQcdNQI1" +
-                  "ATEIIQgdNQQ1AzQBNAMNQAAkNAE0AxI0AjQEDxBAABYAMQkoEjECIQkN" +
-                  "EDEHMgMSEDEIIhIQEA==")
+        golden = ("ASAKAAHAlrECApBOBLlgZB7AxAcmASD+vKC7FEpaTqe0OKRoGsgObKEF" +
+                  "vLYH/FZTJclWlfaiEzEWIhIxECMSEDEBJA4QMgQjEkAAVTIEJRIxCCEE" +
+                  "DRAxCTIDEhAzARAhBRIQMwERIQYSEDMBFCgSEDMBEzIDEhAzARIhBx01" +
+                  "AjUBMQghCB01BDUDNAE0Aw1AACQ0ATQDEjQCNAQPEEAAFgAxCSgSMQIh" +
+                  "CQ0QMQcyAxIQMQgiEhAQ")
         p = s.get_program()
+
         self.assertEqual(p, base64.b64decode(golden))
         self.assertEqual(s.get_address(), golden_addr)
 
         sk = ("DTKVj7KMON3GSWBwMX9McQHtaDDi8SDEBi0bt4rOxlHNRah" +
               "La0zVG+25BDIaHB1dSoIHIsUQ8FFcdnCdKoG+Bg==")
         gh = "f4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGk="
-        [stx_1, stx_2] = s.get_swap_assets_transactions(1000, p, sk, 1234,
-                                                        2234, gh, 10)
-
-        golden_txn_1 = ("gqRsc2lngaFsxLUBIAoAAQoCwMQHBJBOJIECi2omASCSr5GE5B" +
-                        "M8CX9mq8yTsREbEqIBvZ0Myj5KSKHOE8z0xTEWIhIxECMSEDEB" +
-                        "JA4QMgQjEkAAVTIEJRIxCCEEDRAxCTIDEhAzARAhBRIQMwERIQ" +
-                        "YSEDMBFCgSEDMBEzIDEhAzARIhBx01AjUBMQghCB01BDUDNAE0" +
-                        "Aw1AACQ0ATQDEjQCNAQPEEAAFgAxCSgSMQIhCQ0QMQcyAxIQMQ" +
-                        "giEhAQo3R4bomjYW10zIyjZmVlzQiiomZ2zQTSomdoxCB/g7Fl" +
-                        "f/H8U7ktwYFIodZd/C1LH6PWdyhK3dIAEm2QaaNncnDEIBZKtz" +
-                        "WDJ1br7SzgRxVB4GuldnngnHquscPjuK+YxU/homx2zQi6o3Jj" +
-                        "dsQgzUWoS2tM1RvtuQQyGhwdXUqCByLFEPBRXHZwnSqBvgajc2" +
-                        "5kxCCy5lZQqtBjR2Z2VlFOrpSPipyEsUiGthd9feBAjtIc9KR0" +
-                        "eXBlo3BheQ==")
-        golden_txn_2 = ("gqNzaWfEQMvoyBUpU+UgLIRDEOYa7FLhEKUuYWh5esjSqk6ZUm" +
-                        "bSjjGcsv60c18noWTOEIodQ7Rt0gvSqLGCenrnaSs0Nw+jdHhu" +
-                        "iqRhYW10zQPopGFyY3bEIJKvkYTkEzwJf2arzJOxERsSogG9nQ" +
-                        "zKPkpIoc4TzPTFo2ZlZc0JJKJmds0E0qJnaMQgf4OxZX/x/FO5" +
-                        "LcGBSKHWXfwtSx+j1ncoSt3SABJtkGmjZ3JwxCAWSrc1gydW6+" +
-                        "0s4EcVQeBrpXZ54Jx6rrHD47ivmMVP4aJsds0IuqNzbmTEIM1F" +
-                        "qEtrTNUb7bkEMhocHV1KggcixRDwUVx2cJ0qgb4GpHR5cGWlYX" +
-                        "hmZXKkeGFpZM0nEA==")
+        [stx_1, stx_2] = s.get_swap_assets_transactions(
```

Comment:
My only question is, should this return a concatenated byte array like the DynamicFee template?

Reply:
Do you mean for the test? DynamicFee doesn't return a byte array in the Python SDK.

Reply:
This might be a UX question then, I in the other SDKs we format these in a format that could immediately be submitted.

For the test I think it's fine to keep them separate since the goldens can be combined to compare the result either way.

Reply:
The function to submit transactions takes transaction objects so it can be immediately submitted.